sky/packages/sky/lib/src/widgets/scrollable.dart

```diff
@@ -379,6 +405,39 @@ class Block extends StatelessComponent {
   }
 }
 
+abstract class ScrollableListPainter extends Painter {
+  RenderBlockViewport get renderer => renderObject;
```

Comment:
What guarantees (or asserts) that this painter will only be used with RenderBlockViewport?